tests/unit/cluster/atomic-slot-migration.tcl

```diff
@@ -1868,6 +1871,7 @@ start_cluster 3 3 {tags {external:skip cluster} overrides {cluster-node-timeout
        R 3 debug asm-trim-method active 1000000
        R 0 flushall
        populate_slot 100 -slot 0
+       wait_for_ofs_sync [Rn 0] [Rn 3]
```

Comment:
I also fixed this by lowering the per-key sleep time. It wouldn't break activeTrimCycle() loop for 30 seconds otherwise. It seems fixed the flakiness, for the last 10+ runs it didn't fail.

Reply:
since i found a full sync caused the failure, and said replica doesn't have cache master.